translate/cloud-client/src/test/java/com/example/translate/HybridGlossariesIT.java

```diff
@@ -0,0 +1,170 @@
+import com.google.cloud.translate.v3beta1.TranslateTextGlossaryConfig;
+import com.google.cloud.translate.v3beta1.TranslateTextRequest;
+import com.google.cloud.translate.v3beta1.TranslateTextResponse;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient.ListGlossariesPagedResponse;
+
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.EntityAnnotation;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.Feature.Type;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+import com.google.cloud.vision.v1.TextAnnotation;
+
+import com.google.common.html.HtmlEscapers;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.util.JsonFormat;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+
+
+/**
+ * Tests for SsmlAddresses sample.
+ */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class HybridGlossariesIT {
+
+  private static String PROJECT_ID = System.getenv("PROJECT_ID");
+  private static String OUTPUT_FILE = "output.mp3";
+  private static String INPUT_FILE = "resources/test.jpeg";
+  private static String TEXT_FILE = "resources/test.txt";
+  private static String SRC_LANG = "fr";
+  private static String TGT_LANG = "en";
+  private static String GLOSS_NAME = "bistro-glossary";
+  private static String GLOSS_URI =
+      "gs://cloud-samples-data/translation/bistro_glossary.csv";
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+  private File outputFile;
+
+  @Before
+  public void setUp() {
+    // collect project ID from environment
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @Test
+  public void testPicToText() throws Exception {
+    // Act
+    String ocr = HybridGlossaries.picToText(INPUT_FILE);
+
+    // Assert
+    assertThat(ocr).contains("test");
+  }
+
+  @Test
+  public void testCreateGlossary() throws Exception {
+    // Act
+    HybridGlossaries.createGlossary(SRC_LANG, TGT_LANG, PROJECT_ID, GLOSS_NAME, GLOSS_URI);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains("glossary");
```

Comment:
This will never fail as your error message also contains the word 'glossary' instead you should look for `Created: ` along with some output from the glossary.